tests/integration/suite/daprd/state/http/fuzz.go

```diff
@@ -74,7 +75,7 @@ func (f *fuzzstate) Setup(t *testing.T) []framework.Option {
 	fuzzFuncs := []any{
 		func(s *saveReqBinary, c fuzz.Continue) {
 			var ok bool
-			for len(s.Key) == 0 || strings.Contains(s.Key, "||") || ok {
+			for len(s.Key) == 0 || strings.Contains(s.Key, "||") || strings.HasSuffix(s.Key, ".") || ok {
```

Comment:
Again re dots as a suffix. No longer supporting dots at the end of keys would be a breaking change to users. If this is no longer supported, we should have a test case specifically for it, showing it produces a user readable error, and is documented.

Please not this ran hundreds of thousands of times previously with no issue.

Reply:
I responded above.

Reply:
Changed